var/spack/repos/builtin/packages/xsdk-examples/package.py

```diff
@@ -17,6 +17,8 @@ class XsdkExamples(CMakePackage, CudaPackage):
     maintainers("balay", "luszczek", "balos1", "shuds13", "v-dobrev")
 
     version("develop", branch="master")
+    # FIXME: replace the testing branch with the actual release when available:
+    version("0.4.0", branch="testing-v0.4.0")
```

Comment:
Is this going to happen soon? If so, I'd prefer to wait a couple of days and have the sha256 here.

Reply:
The release will happen around mid-May (that's why I added the `WIP` and `don't-merge-yet` labels). The plan is to merge this PR after that. For now, we're just gonna use this branch/PR for testing.